app/modules/edit_content.js

```diff
@@ -8,8 +8,8 @@ export default function editTask() {
       if (e.key === 'Enter') {
         e.preventDefault();
         editedText = todo.innerText;
+        console.log(editedText);
```

Comment:
- [ ] [OPTIONAL] it would be better if you remove the `console.log` from your code it is not a good practice to leave console.log in the production code so please try to remove it